addons/common/functions/fnc_registerAmmoFiredEvent.sqf

```diff
@@ -0,0 +1,54 @@
+/*
+ * Author: esteldunedain
+ * Register an ammo fired event that should be launched when ammo following a
+ * given criteria is fired. The criteria is analyzed on mission start and is
+ * cached.
+ *
+ * Arguments:
+ * 0: Name of the event <STRING>
+ * 1: Condition code <CODE>
+ * 2: Call for the local player on foot? <BOOL>
+ * 3: Call for non local players on foot? <BOOL>
+ * 4: Call for non players on foot? <BOOL>
+ * 5: Call for the local player vehicle? <BOOL>
+ * 6: Call for non local players vehicles? <BOOL>
+ * 7: Call for non players vehicles? <BOOL>
```

Comment:
Maybe change the description to:
```
+ * Arguments:
+ * 0: Name of the event <STRING>
+ * 1: Condition code <CODE>
+ * 2: Call for local player? <BOOL>
+ * 3: Call for remote players? <BOOL>
+ * 4: Call for (local ???) AI soldiers? <BOOL>
```
The rest is still ambiguous. What is a non-local player vehicle? What if the player is the commander?